### PR TITLE
Adding a test plan for QUARKUS-478.

### PR DESCRIPTION
**Pull request by mjurc**

### CODE REVIEW

File: QUARKUS-478.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+Jira: https://issues.redhat.com/browse/QUARKUS-478
+GitHub: https://github.com/quarkusio/quarkus/pull/11760
+
+The goal of QUARKUS-478 is to provide unified experience of build and deploying Quarkus applications to OpenShift. This
+shall be achieved by extending `quarkus-container-image-openshift` extension to enable user to create the build and 
+deployment of application with single, unified command for either JVM and native applications with both classic and 
+serverless application architecture.
+
+## Scope of the testing
```
Comment (rsvoboda) [inline QUARKUS-478.md:11]: I think `quarkus.openshift.jvm-dockerfile=MyDockerFile` and native equivalent should be also covered

File: QUARKUS-478.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+### Impact on resources:
+The new test implemented in Quarkus OpenShift test suite does not require infrastructural changes and its execution
+should be finishined within minutes. Test will be executed on all the tested OCP platforms.
+
+## Getting familiar with the feature
+Following actions were taken to ensure familiarity:
+- Focus on exploratory testing of the features
+- Manual negative testing
+  - Test with no available required dockerfiles
+  - Test with invalid dockerfiles
```
Comment (rsvoboda) [inline QUARKUS-478.md:34]: We checked 
 - corrupted Dockerfile - invalid directory name
 - empty Dockerfile

and `dockerfiles` => `Dockerfiles`

**Review by rsvoboda — CHANGES_REQUESTED** [review]

**Comment by rsvoboda** [comment]
@mjurc will you update the PR anytime soon?

**Comment by mjurc** [comment]
@rsvoboda Done!

**Review by rsvoboda — APPROVED** [review]

**Pull request merged**